src/styles/main.scss

```diff
@@ -0,0 +1,43 @@
+$day-gap: 1px;
+$day-width: 100px;
+
+html {
+  font-family: Arial, sans-serif;
+  font-size: 30px;
+}
+
+body {
+  margin: 0 20px;
+  height: 100vh;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: start;
+  width: 7 * ($day-width + 1);
+  gap: $day-gap;
+  padding: 10px;
+
+  &__day {
+    box-sizing: border-box;
+    background-color: #eee;
+    width: $day-width;
+    height: 100px;
+    border: 1px solid black;
+    gap: $day-gap;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  &__day:hover {
+    background-color: #ffbfcb;
+    transform: translateY(-20px);
+    transition-duration: 0.5s;
+    cursor: pointer;
+  }
```

Comment:
Try to fix animation. Need to be done more smoothly.

Your version:
https://gyazo.com/a8d020baf16fe6ec4beec2ea2b735145

Correct version:
https://gyazo.com/29da9ace4923acc24208029e0df02a69